stuff/cron/aggregate_feedback.py

```diff
@@ -108,8 +138,10 @@ def get_global_quality_and_difficulty_average(dbconn, rank_cutoffs):
                 logging.exception('Failed to parse contents')
                 continue
 
+            before_ac = 'before_ac' in contents and contents['before_ac']
```

Comment:
```suggestion
            before_ac = contents.get('before_ac', 0.0)
```

esto es un poco más conciso.